public/pages/DetectorDetail/utils/constants.ts

```diff
@@ -18,61 +18,59 @@ export enum DETECTOR_DETAIL_TABS {
   CONFIGURATIONS = 'configurations',
 }
 
-const DEFAULT_ACTION_ITEM = 'Please restart this detector to retry.';
+const DEFAULT_ACTION_ITEM = 'Restart the detector and try again.';
 // Known causes:
 // https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/development/src/main/java/com/amazon/opendistroforelasticsearch/ad/transport/AnomalyResultTransportAction.java#L174-L185
 export const DETECTOR_INIT_FAILURES = Object.freeze({
   NO_TRAINING_DATA: {
     //https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/development/src/main/java/com/amazon/opendistroforelasticsearch/ad/transport/AnomalyResultTransportAction.java#L801
     keyword: 'Cannot get training data',
-    cause: 'lack of data ingestion',
+    cause: 'sufficient data is not ingested',
     actionItem:
-      'Please make sure your data ingestion is working. Or increase your detector time interval if data source has infrequent ingestion.',
+      'Make sure your data is ingested correctly. If your data source has infrequent ingestion, increase the detector time interval and try again.',
   },
   COLD_START_ERROR: {
     //https://github.com/opendistro-for-elasticsearch/anomaly-detection/blob/development/src/main/java/com/amazon/opendistroforelasticsearch/ad/transport/AnomalyResultTransportAction.java#L811
     keyword: 'Error while cold start',
-    cause: 'error is found while model initialization',
+    cause: 'of an error during model training',
```

Comment:
There will be some words before this `of`?

Reply:
yes, it is "The detector is not initialized because ". It is [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/pull/95/files#diff-f6c1598a64b7b623fc057e5068b782bfR38).